recipe/meta.yaml

```diff
@@ -1,14 +1,14 @@
 {% set name = "sagemaker-code-editor" %}
-{% set version = "1.2.0" %}
+{% set version = "1.3.0" %}
```

Comment:
how are we managing branches in feedstock repo ? I think we should maintain similar branches as sagemaker-code-editor repo, that'll allow us to provide patches for older minor versions if needed

Reply:
Yup that makes sense to me. I am unable to create branches on this repo, but we can create one for 1.2 and 1.3.